_features/fresh.html

```diff
@@ -2,6 +2,14 @@
 title: Fresh
 slug: fresh
 description: Always up-to-date.
+usecases:
+- As a web application developer, I want to deliver updates seamlessly, without requiring
+  user intervention and approval.
+- As a user, I don't want to wait for the new version to be installed; I always
+  want the latest version available.
 technologies:
 ---
-<p>Progressive Web Apps can work even on lack of network or unreliable connectivity. No more blank connectivity error pages nor dinosaurs running through the desert. A clear separation between UI and content in addition to offline caches and service workers allow you to store the application layout for future use.</p>
+<p>When users visit your web app, they are not prompted with a request to update,
+they simply see yout brand new UI with all your awesome functionality. As web apps, progressive web apps
+behave the same way out of the box but provide you with the means to fully customize the
+life cycle of your app.</p>
```

Comment:
Nit: yout -> your

I still have some general issues with this paragraph, however.

In the first sentence, "they are not prompted with a request to update" suggests an app update has occurred, but you haven't actually said that. If the point is that users aren't prompted after an update, then I would say, "When users visit your web app after it has been updated, they aren't prompted with a request to update…" Otherwise, if the point is that users are *never* prompted with a request to update, because updates are always automatic and silent, then I would say, "When users visit your web app, they're never prompted with a request to update…"

Also, in the first sentence, "see your brand new UI" suggests that updates are UI-specific, which isn't necessarily the case. And "all your awesome functionality" sounds like pandering. I would instead say, "they simply get the updated version of the app" or "they simply get the latest version of the app" or  (depending on whether or not you reference a specific update in the first half of the sentence).

In the second sentence, "behave the same way" is ambiguous: the same way as *what*? And "fully customize the life cycle" is vague. I would clarify this sentence further, although I'm not sure how exactly. Perhaps you can explain your thinking further? If you provide some details about the message you'd like to convey, then I can help distill that into a sentence or two!